.github/pull-request-template.md

```diff
@@ -0,0 +1,21 @@
+Fixes #
```

Comment:
Can you add a comment to explain tot he user to fill this in with the appropriate issue?